reactor-core/src/main/kotlin/reactor/core/publisher/MonoFunctions.kt

```diff
@@ -17,36 +17,17 @@
 package reactor.core.publisher
```

Comment:
is it safe to rename extension classes in Kotlin?

Reply:
I have asked on Kotlin Slack to be sure.